### ISSUE TITLE
Wall Nested Mobs Blocking Directional Targeting

### ISSUE THREAD

**Issue by Superwelder** [cmss13-devs/cmss13#3719]
### Testmerges

#3601, #3645, #3644

### Description of the bug

Any nested mob on a wall is taking priority over non-nested mobs for targeting. This means marines standing in a tile occupied by a nested mob will be immune to directional slashing and tackling as the target of said clicks focuses on the nested mob(often unable to slash since you can't slash infected && nested mobs). This is very likely related to how bullets hit nested people when a xeno occupies the same tile as well.

### What's the difference with what should have happened?

Nested mobs should not be soaking tackles, slashes, or bullets just by virtue of being in the same tile as a non-nested mob.

### How do we reproduce this bug?

1. Nest a mob.
2. Have a marine stand in the nested tile.
3. As xeno with directional slash enabled, try to tackle the marine down using directional targeting.
4. Watch the marine prime grenades or worse inside the hive since they can not be taken down.

### Issue Bingo

- [X] Issue could be reproduced at least once
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)

**Comment by Weuyn** [comment]
oh so this is why I kept missing tackles

**Comment by Superwelder** [comment]
> 

Yes, this bug is particularly dangerous because xeno players often spit out devoured marines in the hive and if they just so happen to spit them out on a tile sharing a nested person, have fun!

**Comment by Wintermote** [comment]
I think this same bug also causes nested marines to absorb bullets going through the tile they're on.
